pynicotine/gtkgui/search.py

```diff
@@ -1415,19 +1415,11 @@ def on_download_folders(self, *_args, download_location=""):
                 visible_files.append(
                     (user, fullpath, destination, size.get_value(), h_bitrate, h_length))
 
-            core.search.request_folder_download(user, folder, visible_files)
-
-    def on_download_folders_to_selected(self, selected, _data):
-        self.on_download_folders(download_location=selected)
-
-    def on_download_folders_to(self, *_args):
+            if (self.download_folder is None):
```

Comment:
We only use parentheses here if the statement is too long to fit on a single line.